src/plugins/projectGalaxy/components/CustomBlock.vue

```diff
@@ -0,0 +1,272 @@
+<script>
+import Plugin from '../index';
+import { useFlashNotification } from '@/composables/useFlashNotification';
+
+const { notify } = useFlashNotification();
+
+const APP_URL = 'https://galaxy.eco';
+const NO_OAT_IMAGE =
+  'https://snapshotsplugin.s3.us-west-2.amazonaws.com/placeholder.png';
+const IMG_LOGO_GALAXY =
+  'https://d257b89266utxb.cloudfront.net/galaxy/images/avatar/0x4960c283c45e1898c41633c39fb2015167b20dc3-1655711973.png';
+const IMG_ICON_LINK =
+  'https://d257b89266utxb.cloudfront.net/galaxy/images/avatar/0x4960c283c45e1898c41633c39fb2015167b20dc3-1655712057.png';
+const IMG_ICONS = {
+  CLAIMED:
+    'https://d257b89266utxb.cloudfront.net/galaxy/images/avatar/0x4960c283c45e1898c41633c39fb2015167b20dc3-1655712157.png',
+  CLAIMING:
+    'https://d257b89266utxb.cloudfront.net/galaxy/images/avatar/0x4960c283c45e1898c41633c39fb2015167b20dc3-1655712194.png'
+};
+
+const STATES = {
+  NO_OAT: {
+    buttonText: 'No OAT'
```

Comment:
Did you consider making your strings translatable? https://docs.snapshot.org/plugins/create#localization

Reply:
We may translate strings late, for now we just focus on english support.